backend/feedback/serializers.py

```
@@ -0,0 +1,38 @@
+            "tension_after",  
+            "helpfulness",  
+            "discomfort_feedback", 
+            "is_delayed",
+            "skipped",
+            "created_at",
+            "evaluated_at",
+        ]
+        read_only_fields = [
+            "id", "intervention_type", "session_id",
+            "is_delayed", "created_at", "evaluated_at",
+        ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)